### ISSUE TITLE
font rendering got blurry

### ISSUE THREAD

**Issue by matthiaskrgr** [jwilm/alacritty#1806]
Linux, x11.

top is the good one, bottom is the fuzzy one of d68ecb0deff8b966a0c7d09203d14bfdf5ccd8ad
![both](https://user-images.githubusercontent.com/476013/48635497-96f26880-e9c8-11e8-8ebf-824c8491b8c9.png)

I didn't do any config changes so I am 99% sure a git commit caused this.

**Comment by matthiaskrgr** [comment]
Bug is caused by  d68ecb0 Add option for dynamic padding (#1780)

**Comment by chrisduerr** [comment]
@matthiaskrgr You're absolutely right. I didn't notice this initially since I rarely use a terminal on my DPI scaled monitor and the font size is fairly big, however a direct comparison makes this pretty obvious.

If it's actually introduced in the option for dynamic padding PR, fixing this should be fairly simple.

**Comment by chrisduerr** [comment]
Note, another reason why I probably haven't noticed this until now is that I am using dynamic padding by default. This only is present when dynamic padding is disabled. I'd assume that the conflict is that the resize logic ignores dynamic padding put the renderer still applies it, or vice-versa. I'll have to take a peek in a minute.

**Comment by chrisduerr** [comment]
This was actually an extremely simple fix related to rounding in the non-dynamic codepath.

As always, testing on as many platforms as possible would be appreciated:
https://github.com/jwilm/alacritty/pull/1807

Should just be good to go though.
